marlowe-dashboard-client/src/Types.purs

```diff
@@ -4,31 +4,35 @@ import Prologue
 
 import Data.Argonaut (Json)
 import Data.Argonaut.Decode (JsonDecodeError)
-import Network.RemoteData (RemoteData)
+import Data.Variant (Variant)
+import Data.Variant as Variant
 import Servant.PureScript (AjaxError)
+import Type.Proxy (Proxy(..))
 
 type JsonAjaxError = AjaxError JsonDecodeError Json
 
+type JsonAjaxErrorRow r = (jsonAjaxError :: AjaxError JsonDecodeError Json | r)
+
+jsonAjaxError :: forall r. JsonAjaxError -> Variant (JsonAjaxErrorRow r)
+jsonAjaxError = Variant.inj (Proxy :: Proxy "jsonAjaxError")
+
+type MetadataNotFoundErrorRow r = (metadataNotFoundError :: Unit | r)
```

Comment:
Should this have the key for the contract for which we failed to find the metadata instead of `Unit`?